docusaurus/docs/Android/01-basics/02-choosing-the-correct-sdk.mdx

```diff
@@ -31,13 +31,14 @@ First, let's take a look into a high level overview of the differences between t
 |  Runtime Customizability   |  ✅ As a custom feature                                  | ✅ As idiomatic Compose code                   |
 |      Development Cost      | ⚠️ Noticeable <br/> (Requires more boilerplate, more files generated or changed)       | ✅ Low impact with fewer files generated or changed |
 |    Stateless Components*    | ⛔️ Main components support only ViewModel driven state.                                | ✅ Fully-independent variants that don't use ViewModels.                               |
-|  Extensive Layout Changes  | ⛔️No out-of-the-box support for replacing whole views.                                 | ✅ Using slot APIs*                                       |
+|  Extensive Layout Changes  | ⛔️No out-of-the-box support for replacing whole views.*                                 | ✅ Using slot APIs*                                       |
 
 - **Main building blocks**: Large elements crucial to a chat experience such as lists, headers and the message composer.
 - **Secondary building blocks**: Smaller elements such as avatars, typing indicators, timestamps etc. Building Composables is a lot
   easier than building Views and this enables us to deliver more reusable components.
 - **Stateless Components**: Components that don't rely on ViewModels and use only pure state, but offer the same behavior as our ViewModel-powered components. These give
 you the option of using your custom data sources to render the data.
+- **Extensive Layout Changes**: Currently in UI Components, `MessageComposerView` is our only `View` based component that allows such changes.
```

Comment:
Did not update the `v5` docs with this since `MessageComposerView` is still experimental in there.